docs/edge-authorizers/deployment-and-operation.mdx

```diff
@@ -56,17 +51,19 @@ aserto control-plane list-connections
 ```
 :::
 
-Then, start the sidcar instance by running the following command. Note that if you turned on decision logging in the configure command, you need to specify the `--data-path` flag, where the DECISION_LOG_DIRECTORY is a path to where the decision logs will be stored.
+Then, start the Authorizer instance by running the following command. Note that if you turned on decision logging in the configure command, you need to specify the `--data-path` flag, where the DECISION_LOG_DIRECTORY is a path to where the decision logs will be stored.
 
 ```sh
 aserto developer start <POLICY_INSTANCE_NAME> [--data-path=<DECISION_LOG_DIRECTORY>]
 ```
 
-Or by using the `docker` CLI selecting the correct image (`ghcr.io/aserto-dev/topaz:latest`):
+### Running the Edge Authorizer using the Docker CLI
 
-```sh
-#!/usr/bin/env bash
+For more advaced scenarios you may want to launch the Authorizer using the `docker` CLI.
+
+The following docker command can be used to run the container. Make sure to update the POLICY_INSTANCE_NAME with your Policy Instance.
```

Comment:
"Make sure to update the POLICY_INSTANCE_NAME with **the name of** your Policy Instance.